### PR TITLE
Add a task for mirroring production CMS content on Staging on a regul…

### PR DESCRIPTION
**Pull request by cadecairos**
…ar interval, while scrubbing any sensitive data in the process

This is pretty difficult to test without setting up a bunch of stuff. I've tested the whole script and scrub end-to-end using a dev app (foundation-staging-clone), using credentials I've generated for the process.
I can guide who ever is reviewing through the testing and provide them with the S3 credentials they need to test it out.

To test, I ran `heroku local:run ./tasks/clone_foundation_site.sh` This loads in a .env file to set config vars, then executes the tasks. There's an `env.dist` with the environment variable names provided already, you only need to add the values.

cc/ @alanmoo @Pomax @gvn

### CODE REVIEW

File: tasks/cleanup.sql [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        new_email := concat(encode(gen_random_bytes(12), 'base64'), '@example.com');
+        new_hash := crypt(encode(gen_random_bytes(32), 'base64'), gen_salt('bf', 6));
+        new_username := concat('anonymouse', counter::varchar);
+
+        UPDATE auth_user
+        SET
+          email = new_email,
+          password = new_hash,
+          username = new_username,
+          first_name = 'anony',
```
Comment (Pomax) [inline tasks/cleanup.sql]: can we make this "Antony"?

File: tasks/clone_foundation_site.sh [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+if [ -x `command -v heroku` ]; then
+    echo "Heroku CLI is already installed..."
+else
+    echo "Downloading and extracting the standalone Heroku CLI tool..."
+    curl "https://cli-assets.heroku.com/heroku-linux-x64.tar.xz" -o "heroku-linux-x64.tar.xz"
+    tar -xf heroku-linux-x64.tar.xz
+    alias heroku=${current_dir}/heroku/bin/heroku
+fi
+
+if [ -x `command -v aws` ]; then
```
Comment (Pomax) [inline tasks/clone_foundation_site.sh]: This did not detect `aws` not being a command, probably worth using `which`

File: tasks/cleanup.sql (hunk 1 above)
Comment (cadecairos) [reply tasks/cleanup.sql]: Sure, why not!

**Comment by cadecairos** [comment]
> When I looked at django_site table, I found "example.com" instead.

I ran the script, and the django_site table is updated as it should be, not sure how you got it in that state

**Comment by cadecairos** [comment]
@patjouk I've fixed the date check like you asked

@Pomax The check for binaries should work properly now.

I also use a trap to interrupt the script on exit and check the exit code. This way I can detect a non-zero exit and rollback staging while ensuring it gets out of maintenance mode and scales up.

File: tasks/clone_foundation_site/task.sh [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+trap 'on_exit' 0
+
+set -e
+
+current_dir=$(pwd)
+
+echo "Checking the day of the week..."
+DAYOFWEEK=$(date +%u)
+if [ "${DAYOFWEEK}" -ne 4 ]; then
```
Comment (patjouk) [inline tasks/clone_foundation_site/task.sh]: Shouldn't it be 1 instead of 4?

File: tasks/clone_foundation_site/task.sh [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+if [ "$?" -eq 0 ]; then
+    echo "Heroku CLI is already installed..."
+else
+    echo "Downloading and extracting the standalone Heroku CLI tool..."
+    curl "https://cli-assets.heroku.com/heroku-linux-x64.tar.xz" -o "heroku-linux-x64.tar.xz"
+    tar -xf heroku-linux-x64.tar.xz
+    alias heroku=${current_dir}/heroku/bin/heroku
+fi
+
+AWS_BIN=$(command -v aws2)
```
Comment (patjouk) [inline tasks/clone_foundation_site/task.sh]: Typo :)

File: tasks/clone_foundation_site/task.sh (hunk 3 above)
Comment (Pomax) [reply tasks/clone_foundation_site/task.sh]: heh, we had set it to 4 because we were testing it on Thursday =P

Comment (patjouk) [reply tasks/clone_foundation_site/task.sh]: I realized the typo when I had to put it to 5 to do testing on Friday XD

**Pull request merged**